pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -554,8 +546,8 @@ func (asw *actualStateOfWorld) MarkFSResizeRequired(
 	}
 }
 
-func (asw *actualStateOfWorld) SetVolumeGloballyMounted(
-	volumeName v1.UniqueVolumeName, globallyMounted bool, devicePath, deviceMountPath string) error {
+func (asw *actualStateOfWorld) setVolumeGloballyMounted(
```

Comment:
I am thinking maybe just remove this setVolumeGloballyMounted function which is confusing to be used in both Mark and UnMark cases. 

In MarkDeviceAsMounted, we set all the paths and mark it as globally mounted. In MarkDeviceAsUnmounted, just set globallyMounted value to false without touching the paths.

Reply:
I had considered that and then decided against it because at least the function preamble is the same. But I agree, it would be cleaner. I'll change it.

Reply:
@jingxu97 done. Is it okay now?